src/components/Modal/Modal.scss

```diff
@@ -1,10 +1,19 @@
 .Modal {
+    &__body--open {
```

Comment:
Having element `body` with modifier `open` doesn't make much sense because if it is in a DOM tree, it is already open (in other words such thing as `body--closed` doesn't exists).

So, can be just `body`.